TRANSLATION_NOTES.asc

```diff
@@ -1,61 +1,54 @@
->>>>>>> 811dffebd4ff1348819f8acc70f954bd4ad65057
+In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit "Du" oder "Sie" angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
 
-=== Übersetzungsfortschritt
+== Übersetzungsfortschritt
 
-Beim Übersetzen sollte zusätzlich zur eigentlichen Übersetzung der Fortschritt jedes Kapitels in der Datei `status.json` hinterlegt werden. Diese Prozentangabe wird auf verschiedenen Seiten verwendet, um die Leser über den Fortschritt der jeweiligen Übersetzung zu informieren. 
+Beim Übersetzen sollte, zusätzlich zur eigentlichen Übersetzung, der Fortschritt jedes Kapitels in der Datei link:./status.json[`status.json`] hinterlegt werden. Diese Prozentangabe wird auf verschiedenen Seiten verwendet, um die Leser über den Fortschritt der jeweiligen Übersetzung zu informieren.
 
-<<<<<<< HEAD
-=== Workflow Git
+== Workflow Git
 
-Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein Fork erstellen und in diesem weiterarbeiten.
+Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein link:https://git-scm.com/book/de/v1/Distribuierte-Arbeit-mit-Git-xxx-An-einem-Projekt-mitarbeiten#Kleine,-öffentliche-Projekte[Fork] erstellen und in diesem weiterarbeiten.
 
-* Bitte erstellen Sie einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request was Ihr zu mergender Branch enthält (Neue Übersetzung, Korrektur, ...).
+* Bitte erstellen Sie erst einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request, was Ihr zu mergender Branch enthält (neue Übersetzungen, Korrekturen, etc.).
 
-* Wir werden Ihr Ergebnis prüfen bzw. ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung damit das Ergebnis insgesamt besser wird und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
+* Wir werden Ihren Beitrag prüfen und ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung, damit das Ergebnis insgesamt besser wird und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
```

Comment:
```suggestion
* Wir werden Ihren Beitrag prüfen und ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung, damit das Ergebnis insgesamt besser wird, und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
```

Comma required here.
Also, the last bit sounds a bit unclear – what are `known sides/pages`?